doc/index.rst

```diff
@@ -93,6 +94,9 @@ debugging, and shortens the round-trip time for testing.
 * `devel jobs#run-the-devel-job-locally <jobs/devel_jobs.rst#run-the-devel-job-locally>`_
   build and test ROS repositories
 
+  * `devel jobs#run-the-devel-job-locally <jobs/devel_jobs.rst#run-the-devel-job-on-travis>`_
```

Comment:
I have updated the PR. Please feel free to apply such trivial changes directly in the future.